tools/insn_freq_explorer.c

```diff
@@ -0,0 +1,265 @@
+/*
```

Comment:
Rename the file name to `tools/rv-histogram`. At the moment, only instruction usage is implemented. Later, we may extend for register usage.